index.d.ts

```diff
@@ -189,6 +189,10 @@ declare module '@primer/components' {
     scheme?: string
   }
 
+  export interface GridProps extends BoxProps, StyledSystem.GridProps, Omit<React.HTMLAttributes<HTMLSpanElement>, 'color'> {}
+
+  export const Grid: React.FunctionComponent<GridProps>
```

Comment:
nit: should we put this above the `CounterLabelProps`? Right now it's separating the `CounterLabelProps` from the `export const CounterLabel...` line.

Reply:
yes good call! This file need some alphabetizing love as well at some point!